src/html/markup.mli

```diff
@@ -19,19 +19,36 @@
 module Html = Tyxml.Html
 
 
-
 val keyword : string -> [> Html_types.span ] Html.elt
 
-module Type : sig
-  val path :
-    [< Html_types.span_content_fun ] Html.elt list ->
+module ML : sig
```

Comment:
I introduced namespaces for Markup... common Markup is still in the toplevel of the module, e.g. `keyword`